lib/simctl.js

```diff
@@ -6,8 +6,7 @@ import _ from 'lodash';
 
 const log = getLogger('simctl');
 
-async function simExec (command:string, timeout:number, args:Array = [],
-    env = {}) {
+async function simCommand (command:string, timeout:number, args:Array = [], env = {}, executingFunction) {
```

Comment:
Does this actually work? Doesn't the linter complain about regular parameters coming after defaulted ones? Even if it doesn't, I don't think we should do it. `executingFunction = _.noop` would suffice.